src/noisepy/seis/plotting_modules.py

```diff
@@ -173,7 +173,7 @@ def plot_substack_cc(sfile, freqmin, freqmax, disp_lag=None, savefig=True, sdir=
         ttr = spair.split("_")
         net1, sta1 = ttr[0].split(".")
         net2, sta2 = ttr[1].split(".")
-        for ipath in path_lists:
+        for ipath in ds.auxiliary_data[spair].list():
```

Comment:
This part is changing in https://github.com/mdenolle/NoisePy/pull/153

Reply:
ok I will revert it back and waiting for #153 to be merged.